proof/refine/RISCV64/Machine_R.thy

```diff
@@ -22,6 +22,31 @@ lemma irq_state_independent_HI[intro!, simp]:
    \<Longrightarrow> irq_state_independent_H P"
   by (simp add: irq_state_independent_H_def)
 
+definition "getCurrentTime_independent_H (P :: kernel_state \<Rightarrow> bool)
+  \<equiv> \<forall>f s. P s \<longrightarrow>
```

Comment:
This is not style conforming. The `definition` line should have a type signature with `where` and the actual definition should start on a separate line.